.vscode/setup.sh

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env bash
+
+#Activate Python
+python -i
```

Comment:
What do you mean by "activate python"?

This command will start python in interactive mode - i.e. so that you can execute python commands like `def my_function():` or `x = 1`. 
There's no reason to have this in your shell script. All of the below commands are shell commands, not python code.

And also, you don't want anything **interactive**. This is meant to be a script that just runs with no user input. If you did want to run python code, you'd have a python script file and then your shell script can execute it with e.g. `python my_python_setup.py`